tutorials/dataframe/df103_NanoAODHiggsAnalysis.C

```diff
@@ -344,7 +345,7 @@ void plot(T sig, T bkg, T data, const std::string &x_label, const std::string &f
    // Canvas and general style options
    gStyle->SetOptStat(0);
    gStyle->SetTextFont(42);
-   auto c = new TCanvas("c", "", 800, 700);
+   auto c = new TCanvas("", "", 800, 700);
```

Comment:
WOW. Something very weird!

If I compile (with g++) the tutorial and have `TCanvas("", "", 800, 700)` --> I will produce 4 histograms as expected.
Same happens if I compile with  `TCanvas("c", "", 800, 700)` (as it was before and was giving warnings).

Now, I instead decided to use the root interpreter. Then I get:
* with `TCanvas("c", "", 800, 700)` gave me the expected results --> 4 histos with actual canvas in them (+warnings).
* with `TCanvas("", "", 800, 700)` produced 4 files, BUT only the first histogram is correct, whereas the others only have title and the "place" where the actual canvas should be is blank!

I cannot spot anything that causes this behaviour in this tutorial. This looks like a bug? @eguiraud @couet